code/modules/mob/mob_grab.dm

```diff
@@ -34,6 +34,7 @@
 	abstract = 1
 	item_state = "nothing"
 	w_class = ITEMSIZE_HUGE
+	destroy_on_drop = TRUE	//VOREStation Edit

```

Comment:
...why was this never done before I wonder?

Reply:
It's not strictly necessary since it has it's own dropped() proc that deletes it, but it does make sense to have it set anyway.